OracleWebCenterContent/dockerfiles/README.md

```diff
@@ -0,0 +1,274 @@
+Extract the downloaded zip files and copy `fmw_12.2.1.4.0_wccontent.jar` file under `dockerfiles/12.2.1.4` .
+Checksum of shiphome binary needs to be mentioned in this [file](12.2.1.4/Checksum). Set the Proxies in the environment before building the image as required, go to directory located at OracleWebCenterContent/dockerfiles/ and run these commands -
+
+```
+#To generate checksum
+md5sum fmw_12.2.1.4.0_wccontent.jar
+
+#To build image
+sh buildDockerImage.sh -v 12.2.1.4.0
+
+#Verify you now have the image
+docker images
+```
+
+# 5. Running Oracle WebCenter Content Container
+ 
+To run the Oracle WebCenter Content container, you need to create:
+* Container to manage the Admin Server.
+* Container to manage the Managed Servers.
+
+## 5.1. Creating containers for WebCenter Content Server
+
+### 5.1.1. Update the environment file 
+
+Create an environment file `webcenter.env.list` file, to define the parameters.
+
+Update the parameters inside `webcenter.env.list` as per your local setup.
+Please note: all parameters mentioned below are manadatory and shouldn't be omitted or, left blank. 
+
+```
+#Database Configuration
+DB_DROP_AND_CREATE=<true or false>
+DB_CONNECTION_STRING=<Hostname/ContainerName>:<Database Port>:<Database Service>
+DB_RCUPREFIX=<RCU Prefix>
+DB_PASSWORD=<Database Password>
+DB_SCHEMA_PASSWORD=<Schema Password>
+
+#configure container
+ADMIN_SERVER_CONTAINER_NAME=<Admin Server Container Name>
+ADMIN_PORT=<Admin Server Port>
+ADMIN_PASSWORD=<Admin Server Password>
+ADMIN_USERNAME=<Admin Server User Name>
+
+DOMAIN_NAME=<domain directory-name>
+UCM_PORT=<port to be used for UCM managed server on container>
+IBR_PORT=<port to be used for IBR managed server on container>
+UCM_HOST_PORT=<host port to access UCM managed server - this is the port value to be used for -p option (left of the colon) sec. ### 5.1.3>
+IBR_HOST_PORT=<host port to access IBR managed server - this is the port value to be used for -p option (left of the colon) sec. ### 5.1.3>
+UCM_INTRADOC_PORT=<UCM intradoc port on container>
+IBR_INTRADOC_PORT=<IBR intradoc port on container>
+
+#component
+component=IPM,Capture,ADFUI
+
+#HOSTNAME
+HOSTNAME=<provide your host name>
+
+#Keep Container alive
+KEEP_CONTAINER_ALIVE=true
+```
+
+### 5.1.2. Admin Container (WCCAdminContainer)
+#### A. Creating and Running Admin Container
+
+Run the following command to create the Admin Server container:
+
+```
+docker run -it --name WCCAdminContainer --network=WCContentNET -p <Any Free Port>:<ADMIN_PORT> -v $DATA_VOLUME:/u01/oracle/user_projects --env-file <PATH_TO_ENV_FILE>/webcenter.env.list oracle/wccontent:12.2.1.4.0
+
+# A sample command will look ike this -
+
+docker run -it --name WCCAdminContainer --network=WCContentNET -p 7001:7001 -v $DATA_VOLUME:/u01/oracle/user_projects --env-file <PATH_TO_ENV_FILE>/webcenter.env.list oracle/wccontent:12.2.1.4.0
```

Comment:
Why are you using `-it` in this sample command? You should be using `-d` or `--detatch` so the container runs in the background.